tests/data_context/test_data_context.py

```diff
@@ -691,156 +693,107 @@ def _validate(  # type: ignore[override]
         }
 
 
-@pytest.mark.xfail(
-    reason="Uses unsupported expectation but tests required behavior - fix this test as part of V1-117"  # noqa: E501
-)
-@pytest.mark.filesystem
-def test_unrendered_and_failed_prescriptive_renderer_behavior(
-    empty_data_context,
-):
-    context = empty_data_context
-
-    expectation_suite_name: str = "test_suite"
-
-    expectation_suite = ExpectationSuite(
-        expectation_suite_name=expectation_suite_name,
-        data_context=context,
-        expectations=[
-            ExpectationConfiguration(type="expect_table_row_count_to_equal", kwargs={"value": 0}),
-        ],
-    )
-    context.suites.add(expectation_suite)
-
-    # Without include_rendered_content set, all legacy rendered_content was None.
-    expectation_suite = context.suites.get(name=expectation_suite_name)
-    assert not any(
-        expectation_configuration.rendered_content
-        for expectation_configuration in expectation_suite.expectation_configurations
-    )
+class TestRenderedContent:
+    @pytest.mark.filesystem
+    def test_no_rendered_content_for_file_data_context(self, empty_data_context: FileDataContext):
+        suite_name = "test_suite"
+        empty_data_context.suites.add(
+            ExpectationSuite(
+                name=suite_name,
+                expectations=[gx.expectations.ExpectTableRowCountToEqual(value=0)],
+            )
+        )
+        expectation_suite = empty_data_context.suites.get(name=suite_name)
 
-    # Once we include_rendered_content, we get rendered_content on each ExpectationConfiguration in the ExpectationSuite.  # noqa: E501
-    expectation_suite = context.suites.get(name=expectation_suite_name)
-    for expectation_configuration in expectation_suite.expectation_configurations:
         assert all(
-            isinstance(rendered_content_block, RenderedAtomicContent)
-            for rendered_content_block in expectation_configuration.rendered_content
+            expectation.rendered_content is None for expectation in expectation_suite.expectations
```

Comment:
It sounds like this wasn't part of this PR, but does the `rendered_content` attribute even need to exist in non-cloud contexts?